frontend/src/components/ParticipantsSearch.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <v-combobox
+    clearable
+    label="Search participants"
+    :items="sortedParticipants"
+    item-text="github"
+    :filter="filterParticipants"
+    v-model="selectedParticipant"
+  ></v-combobox>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+export default {
+  props: {
+    value: {
+      type: Object
+    }
+  },
+  computed: {
+    ...mapGetters('participant', {
+      getParticipants: 'getParticipants'
+    }),
+    sortedParticipants() {
+      const participants = this.getParticipants;
+
+      return participants.sort((a, b) => a.github.localeCompare(b.github));
```

Comment:
If the second would be feasible we could use `first_name` to sort participants instead of github. 
(Knowing someone by name comes naturally and if the pattern I described before is possible, then this additional information would make it easy to distinguish between different people having the same name)